charts/opentelemetry-kube-stack/examples/cloud-demo/rendered/instrumentation.yaml

```diff
@@ -8,9 +8,6 @@ metadata:
     helm.sh/chart: opentelemetry-kube-stack-0.0.8
     app.kubernetes.io/version: "0.103.0"
     app.kubernetes.io/managed-by: Helm    
-  annotations:
-    "helm.sh/hook": post-install,post-upgrade
-    "helm.sh/hook-delete-policy": hook-failed
```

Comment:
These were preventing upgrades from working and im not sure why i had them in the first place!